### DISCUSSION TITLE
Bounding Boxes not Showing anymore for Detections

### DISCUSSION THREAD

**Discussion by Sevyn13** [blakeblackshear/frigate#14652] [Ask A Question]
I think ever since the update where the interface drastically changed I can no longer see the bounding boxes on the detections. How do I see bounding boxes in the new interface? I keep having a person detected on one of my cameras but there is never anyone there and the bounding box would be helpful to see what it is thinking is a person.

**Comment by hawkeye217** [comment]
See the [FAQ](https://github.com/blakeblackshear/frigate/discussions/12724), Usage question 8.
